dear_petition/petition/export/tests/test_cr287.py

```diff
@@ -264,3 +267,16 @@ def test_map_offenses__disposition_date(form, offense1, offense_record1):
     assert form.data["DismissalDate:1"] == utils.format_petition_date(
         offense1.disposed_on
     )
+
+
+#
+# Other
+#
+
+
+def test_checkmark_3b(form, offense1, offense_record1):
+    form.map_additional_forms()
+    assert form.data.get("ChargedA")
+    assert not form.data.get("ChargedB")
+    assert not form.data.get("ChargedDesc")
+    assert not form.data.get("ChargedDescCont")
```

Comment:
Can you add the following testcases?

1. ChargedA and ChargedB are both `False`
2. ChargedA is `True` and ChargedB is `False
3. ChargedB is `True` and `ChargedDesc` is `See addendum`
4. ChargedB is `True` and `ChargedDesc` and `ChargedDescCont` have a record description